include/matx/transforms/reduce.h

```diff
@@ -804,27 +812,32 @@ template <typename T> class reduceOpMax {
 template <typename T> class reduceOpAny {
 public:
   using matx_reduce = bool;
-  __MATX_HOST__ __MATX_DEVICE__ __MATX_INLINE__ T operator()(const T &v1, const T &v2)
+  __MATX_HOST__ __MATX_DEVICE__ __MATX_INLINE__ T Reduce(const T &v1, const T &v2)
   {
     return (v1 != 0) || (v2 != 0);
   }
+  __MATX_HOST__ __MATX_DEVICE__ __MATX_INLINE__ T operator()(const T &v1, const T &v2) { return Reduce(v1, v2); }
   __MATX_HOST__ __MATX_DEVICE__ __MATX_INLINE__ T Init() { return (T)(0); }
+  __MATX_DEVICE__ __MATX_INLINE__ void atomicReduce(T *addr, T val) { atomicAny(addr, val); }
 };
 
 /**
- * Operator for performing an all reduction
+ * Operator for performing an any reduction
  *
- * Performs a reduction of two values of type T by returning 1 if all
+ * Performs a reduction of two values of type T by returning 1 if either
```

Comment:
The new comment doesn't seem right. For an `All` reduction I would except "all" or "both" values are non-zero?

Reply:
Ah, copy paste error.